### PR TITLE
add missing docs examples

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Add missing examples in docs.

### CODE REVIEW

File: docs/resources/service_integration.md [hunk 1]
```diff
@@ -15,7 +15,16 @@ getting metrics from an InfluxDB service to a Grafana service to show dashboards
 Elasticsearch, etc.
 
 ## Example Usage
-{{tffile "examples/resources/aiven_service_integration/resource.tf"}}
+```terraform
```
Comment (mhoffm-aiven) [inline docs/resources/service_integration.md:18]: this should be `terraform fmt`-ed

Comment (byashimov) [reply docs/resources/service_integration.md:18]: It's looks well formatted to me.

Comment (Serpentiel) [reply docs/resources/service_integration.md:18]: looks well formatted to me too

**Review by Serpentiel — DISMISSED** [review]
lgtm

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Pull request merged**